Added files for Facebook-DP-Downloader (#1065)
avinashkranjan/Amazing-Python-Scripts: Facebook-DP-Downloader/Readme.md

# Facebook Profile Picture Downloader

Download the profile picture of any public profile on Facebook by its Facebook id.
<br>

## How it works:

> * Just run the script by entering the range of Facebook id s (valid Facebook-user-id) and get the profile-pictures stored in your local file.

Reviewer: is this still valid?

> # <center> Facebook Profile Picture Downloader </center>

Reviewer: ```suggestion
# Facebook Profile Picture Downloader 
```

> <center> Download the profile picture of any public profile on facebook by just it's fbid. </center>

Reviewer: ```suggestion
<center> Download the profile picture of any public profile on Facebook by its Facebook id. </center>
```

<br>

## Modules used:
---------------
* os

> * request

Reviewer: remove this section and add the requirements.txt file.  Ref this file for example: https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Amazon-Price-Alert/requirements.txt

## How it works:
---------------

> * Just run the script by entering the range of fbids (valid fb-user-id) and get the profile-pictures stored in your local file.

Reviewer: What is this `fbid`? I know the full form but in readme files, this doesn't look good. Replace it with full name: Facebook id for every instance in the readme and the code

> ## Note:

Reviewer: remove this heading, all the instructions below are part of How it works only

--------

> * The fbid is can be thought of as a unique fb-user-id of any profile existing on facebook.

Reviewer: ```suggestion
* The fbid can be thought of as a unique fb-user-id of any profile existing on Facebook.
```
